services/core-api/app/api/mines/project_summary/resources/project_summary.py

```diff
@@ -121,6 +128,10 @@ def put(self, mine_guid, project_summary_guid):
             data.get('documents', []), data.get('contacts', []), data.get('authorizations', []))
 
         project_summary.save()
+
+        if prev_status == 'DFT' and project_summary.status_code == 'OPN':
+            project_summary.send_project_summary_email_to_proponent(mine)
+
```

Comment:
can you add the other email to this check? 

`project_summary.send_project_summary_email_to_ministry(mine)`